local.tf

```diff
@@ -178,6 +183,13 @@ locals {
     "t2.nano"      = false
     "t2.small"     = false
     "t2.xlarge"    = false
+    "t3.2xlarge"   = false
+    "t3.large"     = false
+    "t3.medium"    = false
+    "t3.micro"     = false
+    "t3.nano"      = false
```

Comment:
Maybe don't include nano and micro here if they aren't in the `max_pod_per_node` map also?

Reply:
Agreed. I basically duplicated the t2 block and replaced t2 with t3.

Should I drop the t2.nano and t2.micro from this list also?

Reply:
Ah I see what you did. Yes, I think make your change consistent. Don't worry about the other values.

Reply:
Got it. It's done.

Reply:
@max-rocket-internet 
If you request changes in your "reviews", please try to inspect the code more deeply. This is not a linux kernel repo and your analysis is too superficial. What you do here is actually slowing the progress down.

Reply:
Good catch on these @rochacon and I agree with what you've suggested @max-rocket-internet . I've been experimenting with a way we can start relying on [this utility module](https://github.com/terraform-aws-modules/terraform-aws-ebs-optimized) for the `ebs_optimized` portion and will pitch sourcing it from there.

@strygin - Max's comments here are the furthest thing from pedantic or taste-based. Again, see my feedback elsewhere and prioritize being respectful, thoughtful, and on discussion in future comments.